packages/cli/src/oclif/commands/scaffold.js

```diff
@@ -1,181 +1,115 @@
 const _ = require('lodash');
 const path = require('path');
-const colors = require('colors/safe');
 
 const { flags } = require('@oclif/command');
 
 const BaseCommand = require('../ZapierBaseCommand');
 const { buildFlags } = require('../buildFlags');
+
 const {
-  ensureDir,
-  fileExistsSync,
-  readFile,
-  writeFile
-} = require('../../utils/files');
-const { camelCase, snakeCase } = require('../../utils/misc');
-const { splitFileFromPath } = require('../../utils/string');
-
-// what is the `resources: {}` app definition point?
-const typeMap = {
-  resource: 'resources',
-  trigger: 'triggers',
-  search: 'searches',
-  create: 'creates'
+  plural,
+  updateEntryFile,
+  writeTemplateFile,
+  createTemplateContext
+} = require('../../utils/scaffold');
+
+const getNewFileDirectory = (action, test = false) =>
+  `${test ? 'test/' : ''}${plural(action)}`;
+
+const getLocalFilePath = (directory, noun) => `${directory}/${noun}`;
```

Comment:
Ah, fair point. I would argue that the noun should be `Contact` instead of `New Contact` (and this input's place in the template makes more sense as `contact`), but I realize:

* our examples don't always reflect that
* this should at least work as well as it did before

I'll fix that.